docs/learn/smart-contract-developers/create-nft-collection.md

```diff
@@ -0,0 +1,184 @@
+---
+sidebar_label: '🗃 Create an NFT Collection'
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Create an NFT Collection Using LSP8
+
+This tutorial will explore how to create a collection of unique [digital assets](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md).
+
+:::info
+
+⌨️ The full code of this example can be found in the 👾 [lukso-hardhat-template](https://github.com/CJ42/LUKSO-Hardhat-template).
+
+:::
+
+:::note
+
+This guide builds on top of a Hardhat project using TypeScript as described in the [Getting Started](../smart-contract-developers/getting-started.md) section.
+
+:::
+
+## Setup
+
+Make sure you have the following dependencies installed:
+
+- [`ethers.js`](https://github.com/ethers-io/ethers.js/) (alternatively you can use [`web3.js`](https://github.com/web3/web3.js))
```

Comment:
This is important to clarify.

```suggestion
- [`ethers.js v6`](https://github.com/ethers-io/ethers.js/) (alternatively you can use [`web3.js`](https://github.com/web3/web3.js))
```

Reply:
Thanks. Updated. What is incompatible if v6 is not used?

Reply:
Oh wait, actually it looks like only the `getSigner` from ethers is used. Which is the same between v5 and v6. So I think it should be ok for this code snippet if whether v5 or v6 is used. My bad 🙈

Reply:
no worries, thank you for looking into it